android/build.gradle

```diff
@@ -17,7 +17,7 @@ buildscript {
 rootProject.allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
```

Comment:
It seems to me that we should not **remove** `jcenter()` but rather just put `mavenCentral()` above.
https://onesignal.com/blog/android-migrate-jcenter-to-maven-central/

This also seems to be confirmed by some SO posts that mentions that old packages might still only be present in `jcenter` (https://stackoverflow.com/q/67418153/7967164)

Reply:
Heya, jcenter will be completely shut down on February1, 2022, and at that point, we would have to fully remove it anyway. As such, I feel it best to remove it from now. With regards to the point of old packages being present on jcenter, this repo only uses facebook-android so it should be fine.

Reply:
Good point.